pkg/build/generator/generator.go

```diff
@@ -274,17 +274,33 @@ func (g *BuildGenerator) instantiate(ctx apirequest.Context, request *buildapi.B
 		buildutil.UpdateBuildEnv(newBuild, request.Env)
 	}
 
-	// Update the Docker build args
+	// Update the Docker strategy options
```

Comment:
Deserves a unit test in this package proving that leaving the value nil on build request results in the original value being chosen (i.e. true on the build config and nil on the build request).  This is easy to get wrong or regress without noticing.

Reply:
Test(s) added, and also fixed one that wasn't testing what it should have been (thanks @csrwng for the help!)